### ISSUE TITLE
Nice fucking virus lol 

### ISSUE THREAD

**Issue by AngryJosh24** [hydy100/R3nzSkin#172]
This is a virus, it will steal all your passwords on literally every platform

**Comment by 0xJsoul** [comment]
How come you're the only one here that got hacked

and why create another alt account to just say it's a virus again? just get a life, or you can't.

@hydy100 just don't entertain people like this and close this. they are riot fans that don't want people using skins for free

**Comment by AngryJosh24** [comment]
> How come you're the only one here that got hacked
> 
> and why create another alt account to just say it's a virus again? just get a life, or you can't.
> 
> @hydy100 just don't entertain people like this and close this. they are riot fans that don't want people using skins for free

wtf are u waffln about dawg I never had a github account and I created one just so I can write this. the one that I downloaded which was 14.17 one, got almost all the accounts I have got hacked, and no they did not share same passwords. After I used anti virus and completely remove the app all of sudden I'm back to not getting hacked again. If other people were saying it is virus then it must be fucking true because I ain't using alts dumass

**Comment by AngryJosh24** [comment]
> How come you're the only one here that got hacked
> 
> and why create another alt account to just say it's a virus again? just get a life, or you can't.
> 
> @hydy100 just don't entertain people like this and close this. they are riot fans that don't want people using skins for free

u make zero fucking sense lmao ALL I want is to use free skins WITHOUT the damn viruses

**Comment by AngryJosh24** [comment]
> Once doubt arises, it seems that facts don't matter to you anymore. You seem convinced that I hacked your account but can't provide any valid evidence (the program has always been there, and you haven't even bothered to check the detailed report to confirm if it has any network activity). I don't know what else to say, brother.

Oh really that totally explains why literally all the accounts that I had saved on PC got hacked huh, I've never in my life once had my accounts hacked. Untill I started using your injector, oh and it magically stops when I remove it with anti virus. That makes total sense, it seems like you are confusing me with someone else because this account is new, well this app must really be a virus if others are saying that too

**Comment by 0xJsoul** [comment]
'others', it's only 1 guy and you brother

**Comment by AngryJosh24** [comment]
> 'others', it's only 1 guy and you brother

yeah if I was trynna hack other people via sharing an app I wouldn't hack everybody that gets the app, and it doesn't matter two of us still got hacked. The owner says "run the network diagnosis" but that doesn't fucking rule out the local malicious activity lol I can get you screenshots from my antivirus too

**Comment by Eff0rtly** [comment]
> > 'others', it's only 1 guy and you brother'其他人'，只有一个人和你哥哥
> 
> yeah if I was trynna hack other people via sharing an app I wouldn't hack everybody that gets the app, and it doesn't matter two of us still got hacked. The owner says "run the network diagnosis" but that doesn't fucking rule out the local malicious activity lol I can get you screenshots from my antivirus too是的，如果我试图通过分享应用程序来入侵其他人，我不会入侵所有获得该应用程序的人，而且我们两个仍然被黑客入侵也没关系。所有者说“运行网络诊断”，但这并不能排除本地的恶意活动，哈哈，我也可以从我的防病毒软件中获取您的屏幕截图

"Then show the evidence instead of constantly complaining, otherwise it's just slander."

**Comment by Eff0rtly** [comment]
> > 'others', it's only 1 guy and you brother
> 
> yeah if I was trynna hack other people via sharing an app I wouldn't hack everybody that gets the app, and it doesn't matter two of us still got hacked. The owner says "run the network diagnosis" but that doesn't fucking rule out the local malicious activity lol I can get you screenshots from my antivirus too

"Then show the evidence instead of constantly complaining, otherwise it's just slander."

**Comment by 0xJsoul** [comment]
yeah, none of these are true, the anti virus or virustotal detected it as malicious and he goes crazy lol

**Comment by AngryJosh24** [comment]
> yeah, none of these are true, the anti virus or virustotal detected it as malicious and he goes crazy lol

what isn't true? are u saying that network diagnosis can detect local activities too? I just explained you all of my accounts that were on my PC got hacked, but I guess you cant read cuz Im apparently freaking out cuz of virustotal lmao

**Comment by 0xJsoul** [comment]
😂

**Comment by AngryJosh24** [comment]
![Screenshot 2024-10-12 000207](https://github.com/user-attachments/assets/3a6a7333-f97d-4cc3-b910-6851a6bd89ca)

**Comment by Eff0rtly** [comment]
> ![Screenshot 2024-10-12 000207](https://private-user-images.githubusercontent.com/184700488/375934705-3a6a7333-f97d-4cc3-b910-6851a6bd89ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MTY5NzEsIm5iZiI6MTcyODcxNjY3MSwicGF0aCI6Ii8xODQ3MDA0ODgvMzc1OTM0NzA1LTNhNmE3MzMzLWY5N2QtNGNjMy1iOTEwLTY4NTFhNmJkODljYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMlQwNzA0MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDM0NjA0ZDRhMjM2MWViNmRkZTZmMWM3MDdiNzY3NGFiYWE1MDIwNzNjM2Y3MDVkMTExODI0ZDlkYjFlYzk2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JuWyyz977p4tCR0e3OMJ1Woda6kwUgMVcnrfN4fwUT0)

Aren't you at all suspicious of the file with 6.3.5?

**Comment by AngryJosh24** [comment]
> > ![Screenshot 2024-10-12 000207](https://private-user-images.githubusercontent.com/184700488/375934705-3a6a7333-f97d-4cc3-b910-6851a6bd89ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MTY5NzEsIm5iZiI6MTcyODcxNjY3MSwicGF0aCI6Ii8xODQ3MDA0ODgvMzc1OTM0NzA1LTNhNmE3MzMzLWY5N2QtNGNjMy1iOTEwLTY4NTFhNmJkODljYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMlQwNzA0MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDM0NjA0ZDRhMjM2MWViNmRkZTZmMWM3MDdiNzY3NGFiYWE1MDIwNzNjM2Y3MDVkMTExODI0ZDlkYjFlYzk2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JuWyyz977p4tCR0e3OMJ1Woda6kwUgMVcnrfN4fwUT0)
> 
> Aren't you at all suspicious of the file with 6.3.5?

I mean I guess, I already removed all of them. Are you sure this skin changer isn't a virus? How can I tell?

**Comment by 0xJsoul** [comment]
no, you're a retard don't use it

**Comment by AngryJosh24** [comment]
> no, you're a retard don't use it
> 
> pls for the love of god

that makes total sense and helps a lot keep spreading ur virus then edgy ass illiterate no life pos

**Comment by hcy0317** [comment]
> > yeah, none of these are true, the anti virus or virustotal detected it as malicious and he goes crazy lol是的，这些都不是真的，防病毒软件或 VirusTotal 检测到它是恶意的，他发疯了，哈哈
> 
> what isn't true? are u saying that network diagnosis can detect local activities too? I just explained you all of my accounts that were on my PC got hacked, but I guess you cant read cuz Im apparently freaking out cuz of virustotal lmao什么不是真的？您是说网络诊断也可以检测本地活动吗？我刚刚向你解释了我电脑上的所有帐户都被黑客入侵了，但我想你看不懂，因为我显然因为 virustotal lmao 吓坏了

LOL, if this software doesn't have any network activity, how exactly is it supposed to steal your account and password and send them to the hacker's computer? Maybe I'm wasting my time explaining this to you, given that your pea-sized brain probably can't grasp it.

**Comment by Bullseye110** [comment]
> no, you're a retard don't use it
> 
> pls for the love of god

Just listen this man of culture and get out of here, noob. You have more files flagged by your joke antivirus and you only think that this skin changer is the problem.
Get out of here and go to pay for rito skins!

**Comment by rangigo** [comment]
This guy just rage baits and yapping without any proof. You should close this
